packages/reactive-element/src/css-tag.ts

```diff
@@ -172,9 +172,12 @@ export const adoptStyles = (
   styles: Array<CSSResultOrNative>
 ) => {
   if (supportsAdoptingStyleSheets) {
```

Comment:
Factor this into `adoptStyleSheets` and return true if it actually adopts the sheets: L174-180

Reply:
Will do.